crates/extensions/tedge_config_manager/Cargo.toml

```diff
@@ -25,6 +25,7 @@ tedge_mqtt_ext = { workspace = true }
 tedge_uploader_ext = { workspace = true }
 tedge_utils = { workspace = true }
 thiserror = { workspace = true }
+tokio = { workspace = true }
```

Comment:
I'm surprised to see tokio is added here as a part of this PR. What tool suggested to add this?

Reply:
Actually this may not be required anymore. It was previously required due to a patch that @jarhodes314 provided me...but I think the patch was reverted as it was no longer needed, but this change remained. I will revert this change.

Reply:
Removed unused dependency e4cf7f761b624857fdcd5c34e88ad7a1a39d1240